### PR TITLE
Add ee webhooks to celery paths

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
*If this affects the front-end, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, Team, and User (if this PR affects ANY querysets/queries).
- [ ] Django backend tests (if this PR affects the backend).
- [ ] Cypress end-to-end tests (if this PR affects the frontend).

### CODE REVIEW

File: posthog/settings.py [hunk 1]
```diff
@@ -370,7 +370,7 @@ def print_warning(warning_lines: Sequence[str]):
 # NB! This is set to explicitly exclude the "posthog-plugins" queue, handled by a nodejs process
 CELERY_QUEUES = (Queue("celery", Exchange("celery"), "celery"),)
 CELERY_DEFAULT_QUEUE = "celery"
-CELERY_IMPORTS = ["posthog.tasks.webhooks"]  # required to avoid circular import
```
Comment (timgl) [inline posthog/settings.py:373]: ee might not be available

Comment (EDsCODE) [reply posthog/settings.py:373]: Yup

File: posthog/settings.py [hunk 2]
```diff
@@ -371,6 +371,10 @@ def print_warning(warning_lines: Sequence[str]):
 CELERY_QUEUES = (Queue("celery", Exchange("celery"), "celery"),)
 CELERY_DEFAULT_QUEUE = "celery"
 CELERY_IMPORTS = ["posthog.tasks.webhooks"]  # required to avoid circular import
+
+if PRIMARY_DB == CLICKHOUSE:
```
Comment (EDsCODE) [inline posthog/settings.py:375]: Or we could just to a try import to test if it exists and then append?

Comment (timgl) [reply posthog/settings.py:375]: Yeah we already do that on line 234 and 499

**Pull request merged**